types/types.go

```diff
@@ -78,14 +78,14 @@ func (m *ComMatrix) String() string {
 }
 
 func (cd ComDetails) String() string {
-	return fmt.Sprintf("%s,%s,%s,%s,%s,%s,%s,%s,%v", cd.Direction, cd.Protocol, cd.Port, cd.Namespace, cd.Service, cd.Pod, cd.Container, cd.NodeRole, cd.Optional)
+	return fmt.Sprintf("%s,%s,%d,%s,%s,%s,%s,%s,%v", cd.Direction, cd.Protocol, cd.Port, cd.Namespace, cd.Service, cd.Pod, cd.Container, cd.NodeRole, cd.Optional)
 }
 
 func RemoveDups(outPuts []ComDetails) []ComDetails {
```

Comment:
nit: it seems this and the `CmpComDetails` are called together, wdyt about just combining them into one function that remove dups and sorts?